### DISCUSSION TITLE
VisualStudio 2022

### DISCUSSION THREAD

**Discussion by sfullerbeckman** [wixtoolset/issues#6646] [Questions] [answered]
When is the VisualStudio 2022 extension going to be released to the Wixtoolset.org websight? I need this so that I can start developing Wix project in VisualStudio 2022?

**Comment by robmen** [comment, marked as answer]
A pre-release is available at [wixtoolset/VisualStudioExtension](https://github.com/wixtoolset/VisualStudioExtension/). Another fix may be necessary to [resolve a GitHub Actions issue](https://github.com/actions/virtual-environments/issues/4419).
